build-config/make/openssl.make

```diff
@@ -31,10 +31,26 @@ PHONY += openssl openssl-download openssl-source \
 	 openssl-configure openssl-build openssl-install openssl-clean \
 	 openssl-download-clean
 
+ifeq ($(OPENSSL_VERSION),1.1.1g)
+OPENSSL_ARCH	=
 OPENSSL_LIBS	= \
 	engines-1.1 \
 	libcrypto.so libcrypto.so.1.1 \
 	libssl.so libssl.so.1.1
+else ifeq ($(OPENSSL_VERSION),3.4.0)
+ifeq ($(ARCH),arm64)
+OPENSSL_ARCH	= linux-aarch64
+else
+OPENSSL_ARCH	= linux-$(ARCH)
+endif
+
+OPENSSL_LIBS	= \
+	engines \
```

Comment:
Is it engines, not engines-3?
Did you check the build for some x86-64 arch system?

Reply:
There is no need to check it on a physical machine, 
ensure that x86-64 build is not broken with new OpenSSL version.
I remember that OpenSSL 3.0.13 build for x86-64 required engines-3.